src/sidebar/components/annotation-publish-control.js

```diff
@@ -58,9 +58,7 @@ function AnnotationPublishControl({
       <div
         className="annotation-publish-control__btn-dropdown-arrow-indicator"
         style={applyTheme(themeProps, settings)}
-      >
-        <div>▼</div>
```

Comment:
I noticed that ChromeVox screen reader was reading out this char as "down triangle" and in Mac voice over it was "Down pointy black point button" or something like that, quite funny! So I moved it to css ::after so  now is correctly reads out "Change annotation sharing setting" along with menu's name as its collapsed state.